bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/dtree/inst/Graces.scala

```diff
@@ -22,4 +21,19 @@ object Graces {
     }
   }
 
+  class ReadNode(fm: GracesFiberMode)  extends SingleSelectNode[GracesFiberMode, GracesReadMode, GracesBlueprint](fm) {
```

Comment:
Actually it refers to the Node on the decision tree, like the previous one is called `FiberNode`
Perhaps a better name is `ReadModeNode`